_drafts/SC3-1-2-lang.md

```diff
@@ -0,0 +1,76 @@
+---
+name: Validity of HTML Attribute with in elements of page body
+
+description:
+- This rule checks that `lang` and/or `xml:lang` attribute on elements with in the `body` of a web page are correct.
+
+success_criterion:
+- 3.1.2
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Bryn Anderson
+- Jey Nandakumar
+---
+
+## Test Procedure
+
+### Applicability
+
+- Any DOM element, other than the root DOM node with a `lang` and/or `xml:lang` attribute.
```

Comment:
Is it in fact any DOM element or just the ones that are e.g. exposed to assistive technologies that should be applicable to this rule?